Ka'het: "Cloud Probe" and "Lightning in a Bottle" missions (#12386)
endless-sky/endless-sky: data/kahet/kahet missions.txt

event "ka'het: alix discoveries"
	planet Alix
		add description `	Long before becoming an automated mining outpost, this moon was designated as a Builder quarantine site.`

mission "Ka'het: Cloud Probe 1"
	name "In the deep"
	description "Escort a Swan with four modified Terns through a number of gas giants, starting from <destination>."
	source
		government "Remnant"
		attributes "remnant primary"
	destination "Crataegusa"
	to offer
		has "Ka'het: Patir Mystery 36: done"
		random < 30
	on offer
		conversation

> 			`A glance around the landing zone spots two Remnant walking at a fast pace toward your ship; eventually, you recognize them as Taely, still wearing her flight helmet, and Torza. "We have been looking for you, Captain," he begins. He gestures toward the ship they arrived from: a Swan, carrying four Terns below its massive wings. "Have you been in the Graveyard recently?"`

Reviewer: He asked whether you've recently visited the Graveyard, but you can't answer.

			choice
				`	"Will that hide them from the Ka'het?"`
				`	"Why the Swan?"`
					goto swan
			`	"In our experience, it will not," Torza replies, sharing a somber glance with the other prefect. "But the Ka'het are not gaslining, as far as we could tell. If it studies them, they will know it is there, but it should be far beyond their reach."`
			label swan
			`	Taely takes the word. "You might have noticed that neither of our carriers are capable of cloaking. Even if filled with propellant, an Ibis would have to stop, vulnerable, to refuel in the middle of the Graveyard; the Swan is inherently capable of doing the round trip without a ramscoop, using a Scram drive, and leaving some for an escort. Which would be you, in this case."`
			choice
				`	"I would be happy to."`
				`	"Sorry, but I am too busy right now."`
					decline
			`	"Make sure to use a gaslining ship," she reminds you. "Your first target will be in <system>, on the opposite edge of the Ember Waste. The Graveyard is host to gas giants strangely different from the ones here; only <planet> is similar enough for us to trial it before entering the battlefield."`

> 			`	"Be careful with our Swan," adds Torza. "It is meant to support a fleet in combat, not a one-on-one fight. If you encounter trouble, better use some of its fuel reserves to jump somewhere else."`

Reviewer: ```suggestion
			`	"Be careful with our Swan," adds Torza. "It is meant to support a fleet in combat, not win a one-on-one fight. If you encounter trouble, better use some of its fuel reserves to jump somewhere else."`
```

		personality escort unconstrained launching

mission "Ka'het: Cloud Probe 3"
	landing
	name "In the deep"
	description "Reach <destination> with the Swan to deploy the third Tern."
	source "Antamesa"
	destination "Baliesa"
	to offer
		has "Ka'het: Cloud Probe 2: done"
	on complete
		dialog "Another Tern drops from its mothership, falling through the bright clouds. Some of the orbiting debris enters the atmosphere around the same time, leaving a streaks of plasma around the trail of the Tern. While you glance at the spectacle, the map signals you the next stop - Meligara, part of the Ritilas system."
	npc accompany save
		government "Remnant"
		ship "Swan" "Serene Pillar Lost"
		ship "Petrel" "Sunlit Song Remade"
		ship "Petrel" "Sharp Sky Restored"
		personality escort unconstrained launching
	npc
		government "Ka'het"
		system "Gerenus"
		personality staying disables plunders
		fleet
			names "ka'het"
			variant
				"Maeri'het (Disable-able)"

mission "Ka'het: Cloud Probe 4"
	landing
	name "In the deep"
	description "Bring the Swan with the final Tern to <destination>."
	source "Baliesa"
	destination "Meligara"
	to offer
		has "Ka'het: Cloud Probe 3: done"
	on complete

> 		dialog "While the final Tern disappears beyond your sight, you receive a message from the Swan captain. With this task complete, they ask you to escort them to Seraglio - a small planet of the Arculus system - before you part ways. When you inquire further, she simply states that they have been requested there next."

Author: Changed

			choice
				`	"What is your mission here?"`
					goto mission
				`	(Return the farewell.)`
					decline
				`	"What should I do now?"`
			`	A cheerful chant comes out of your radio. It speaks of the unpredictability of life, of the endless paths one may find, and of a hidden purpose unknown to us all. At least three or four voices complement each other, increasing further when the subject changes to the need to take action and not let life slide off you.`
			`	After a few minutes, you infer that they do not know about the mission, and you go your own way. Soon enough, the screen informs you of a transfer of <payment> from Remnant Logistics coupled with a short message of appreciation.`
				decline
			label mission
			`	You are surprised to hear a loud, triumphant chant come out of your radio. It describes the bliss of finding life in unexpected places, beyond what one would consider possible. While the chorus has a very pleasing rhythm, you are oblivious to most of the events described in the song, excluding the discovery of past life in the Sol system.`

> 			`	By the time the song comes to an end, the Swan has completed the landing, and a flexible tube is coming out of its wings. You bid the crew farewell and go your own way. Soon enough, the screen in front of you informs you of the transfer of <payment> from Remnant Logistics, coupled with a short message of appreciation.`

Reviewer: I was assuming the player would be somewhat close, since they bid the crew goodbye

			`	"This has multiple uses. As in this case, we can stay awake for as long as it takes for an emergency to get under control without excess fatigue. Alternatively, we can use it to adapt to different planets, where the length of the day-night is sometimes wildly different. I recall from your records that there are medicines used for similar purpose, but we prefer this approach. Though," she smiles, "I have been told that it can give a tendency to digress."`
			label meaning
			`	Taely walks toward one of your screens, eventually making it show the data of one of the Terns you have recently deployed. "Ever since the first ion storm, the probe stationed in Meligara has become useless. Every tracker returns vast amounts of noise, so much that it is impossible to distinguish a Ka'het in that mess. If, as we suspect, there is a gap in its ion shielding, it is only a matter of time until we lose it. Today marks the fifth day we have spent trying to identify the issue."`

> 			`	She starts tracing lines by hand around a few sections of the Tern. "Each of these areas could be the cause of the problem. If you make a section of these parts with your equipment, we should, at least, prevent the other probes from falling to the same issue."`

Reviewer: > section of these parts with your equipment

I don't follow.

			choice
				`	"Is that one of yours?"`
				`	(Warm up my weapons.)`
			`	Right as the object disappears, a three-letter reply finally reaches you through the radio: "E. M. P."`
			`	The electric burst hits you a few seconds later; it seems to have dissipated somewhat, but it is enough to temporarily garble your communicator. A check of the tracking screen shows the Tern unscathed by the strike, but the aft of the Builder probe is now showing a worrying red glow.`
			`	You try to contact the Remnant, but the arrival of another EMP prevents you from doing so. For their part, one of the Peregrines has taken the initiative to approach the alien spacecraft.`
			choice
				`	(Return to orbit to fight the Ka'het.)`
					flee
				`	(Keep trying to speak with the Remnant.)`
			action
				set "ka'het: cloud probe electronics burnt"
			`	The Peregrine pitches up to reveal two enormous claws mounted beneath its body. It closes the distance with the target bit by bit, then makes a swift movement to catch its prey before the wind changes. It succeeds, despite the continuous lightnings that now surround you.`

> 			`	Once the second Peregrine repeats the movement, a group of Remnant in extravehicular attire comes out of their payload bays. Half of them works on securing the lower section of the craft, most of which is now in a bright, yellow hue; at the same time, you see the other few distribute along the balloon's various tethers. You see a flash: the balloon flies away free, while the payload stays into the clamps of the Peregrines.`

Reviewer: ```suggestion
			`	Once the second Peregrine repeats the movement, a group of Remnant in extravehicular attire comes out of their payload bays. Half of them work on securing the lower section of the craft, most of which is now a bright, yellow hue; at the same time, you see the other few distribute along the balloon's various tethers. You see a flash: the balloon flies away free, while the payload stays in the clamps of the Peregrines.`
```

event "ka'het: alix discoveries"
	planet Alix
		add description `	Long before becoming an automated mining outpost, this moon was designated as a Builder quarantine site.`

mission "Ka'het: Cloud Probe 1"
	name "In the deep"

> 	description "Escort a Swan with four modified Terns through a number of gas giants, starting from <destination>."

Reviewer 2: I think it's clearer if the description is in the same order as the tasks you need to do. So maybe:

```suggestion
	description "Escort a Swan with four modified Terns from <destination> through a number of gas giants."
```

or

```suggestion
	description "Escort a Swan with four modified Terns through a number of gas giants before returning to <destination>."
```

You could also maybe do something like "through the upper atmosphere of a number of gas giants"

	source
		government "Remnant"
		attributes "remnant primary"
	destination "Crataegusa"
	to offer
		has "Ka'het: Patir Mystery 36: done"
		random < 30
	on offer
		conversation

> 			`A glance around the landing zone spots two Remnant walking at a fast pace toward your ship; eventually, you recognize them as Taely, still wearing her flight helmet, and Torza. "We have been looking for you, Captain," he begins. He gestures toward the ship they arrived from: a Swan, carrying four Terns below its massive wings. "Have you been in the Graveyard recently?"`

Reviewer 2: Can a glance spot something, or do you spot something when glancing?

```suggestion
			`Glancing around the landing zone, you spot two Remnant walking at a fast pace toward your ship; eventually, you recognize them as Taely, still wearing her flight helmet, and Torza. "We have been looking for you, Captain," he begins. He gestures toward the ship they arrived from: a Swan, carrying four Terns below its massive wings. "Have you been in the Graveyard recently?"`
```

			`	The three of you take a seat in a starport bench. Torza seems to be concerned about the amount of data that they possess on the inhabitants in the Graveyard. "Our current surveillance methods are limited," he continues, "especially around the Ka'het. One must stay cloaked at all times in order to survive." Taely intervenes with some technical details - of the compromises between ramscoop requirements and designing for optimal cloaking, and how running them continuously will inevitably clog them over time. While you do not recognize every sign she makes, the exploding motion she makes with her hands makes the consequences clear.`

> 			`	"But if we cannot make them undetectable, perhaps we can make them unreachable. We have recently learned how to replicate extremely precise trackers," she states, "enough to record orbital vehicles from deep inside an atmosphere. The idea became to take a Tern, mount these optics inside of it, and leave it permanently inside a Graveyard gas giant."`

Reviewer 2: This might just be me, but I feel like "idea became" is a little clunky? Maybe "idea arose" or "idea was raised" would be better?

```suggestion
			`	"But if we cannot make them undetectable, perhaps we can make them unreachable. We have recently learned how to replicate extremely precise trackers," she states, "enough to record orbital vehicles from deep inside an atmosphere. The idea was raised to take a Tern, mount these optics inside of it, and leave it permanently inside a Graveyard gas giant."`
```

			choice
				`	"Will that hide them from the Ka'het?"`
				`	"Why was that not a concern previously?"`
					goto concern
				`	"Why the Swan?"`
					goto swan
			`	"In our experience, it will not," Torza replies, sharing a somber glance with the other prefect. "But the Ka'het cannot visit gas planets, as far as we can tell. If they study them, they will learn of the Terns, but the ships should be far beyond their reach."`
				goto swan
			label concern
			`	"It has been a point of concern for decades now." Torza looks slightly miffed by your question. "If this information gap is still unresolved, it is only because our attempts at filling it have failed."`
			label swan

> 			`	Taely continues the conversation. "You might have noticed that neither of our carriers are capable of cloaking. Even if filled with propellant, an Ibis would have to stop, vulnerable, to refuel in the middle of the Graveyard; the Swan is inherently capable of doing the round trip without a ramscoop, using a Scram drive, and leaving some fuel for an escort. Which would be you, in this case."`

Reviewer 2: ```suggestion
			`	Taely continues the conversation. "You might have noticed that neither of our carriers are capable of cloaking. Even if filled with propellant, an Ibis would have to stop, vulnerable, to refuel in the middle of the Graveyard; the Swan is inherently capable of doing the round trip without a ramscoop, using a Scram drive and leaving some fuel for an escort. Which would be you, in this case."`
```

Maybe? Because I feel like the two later clauses are qualifying the first one rather than adding to a list. Could go either way, though.

			choice
				`	"I would be happy to."`
				`	"Sorry, but I am too busy right now."`

> 					decline

Reviewer 3: Can this be deferred instead of declined?

Author: It's possible, but I would rather not add a defer after so much text

	npc
		government "Ka'het"
		system "Gerenus"
		personality staying disables plunders
		fleet
			names "ka'het"
			variant
				"Maeri'het (Disable-able)"

mission "Ka'het: Cloud Probe 4"
	landing
	name "In the deep"
	description "Bring the Swan with the final Tern to <destination>."
	source "Baliesa"
	destination "Meligara"
	to offer
		has "Ka'het: Cloud Probe 3: done"
	on complete
		dialog "While the final Tern disappears beyond your sight, you receive a message from the Swan captain. With this task complete, they ask you to escort them to Seraglio - a small planet of the Arculus system - before you part ways. When you inquire further, she simply states that they have been assigned there next."
	npc accompany save
		government "Remnant"
		ship "Swan" "Serene Pillar Lost"
		ship "Tern" "Sunlit Song Remade"
		personality escort unconstrained launching

mission "Ka'het: Cloud Probe 5"
	landing
	name "In the deep"
	description "Escort the Swan and its crew to <destination> to conclude the mission."
	source "Meligara"
	destination "Seraglio"
	to offer
		has "Ka'het: Cloud Probe 4: done"
	on complete
		payment 350000
		event "ka'het: gaslining probe storm" 3
		conversation

> 			`Despite the improved safety of the Remnant cluster, the now-empty Swan silently stays on your wing during the descent. As you fly in the direction of the equator, the ice fields below you leave space to some patches of peculiar, deep red terrain. Occasionally, a dark Remnant outpost adorns the scene, surrounded by a perfect circle of melted ice. It is far too small, and you are moving far too fast, to make out any detail.`

Reviewer 4: ```suggestion
			`Despite the improved safety of the Remnant cluster, the now-empty Swan silently stays on your wing during the descent. As you fly in the direction of the equator, the ice fields below you leave space to some patches of peculiar, deep red terrain. Occasionally, a dark Remnant outpost adorns the scene, surrounded by a perfect circle of melted ice. They are far too small, and you are moving far too fast, to make out any detail.`
```
Also, I feel like 'leave space to' is a bit clunky. Maybe 'give way to' or 'leave space for'?

> 			`	You start to decelerate near your destination, but the Swan passes in front of you without appearing to slow down. Only when it is right above the target - another outpost, seemingly identical to every other one around here - it finally pitches up its nose, flexing its wings to maintain the balance and rotating its nose at the same time. If it is really less 'alive' than other recent Remnant vehicles, you are having a hard time finding differences.`

Reviewer 2: ```suggestion
			`	You start to decelerate near your destination, but the Swan passes in front of you without appearing to slow down. Only when it is right above the target - another outpost, seemingly identical to every other one around here - it finally pitches up its nose, flexing its wings to maintain the balance and rotating its nose at the same time. If it is really less "alive" than other recent Remnant vehicles, you are having a hard time finding differences.`
```

American English

				decline
			label mission
			`	You are surprised to hear a loud, triumphant chant come out of your radio. It describes the bliss of finding life in unexpected places, beyond what one would consider possible. While the chorus has a very pleasing rhythm, you are oblivious to most of the events described in the song, excluding the discovery of past life in the Sol system.`
			`	By the time the song comes to an end, the Swan has completed the landing, and a flexible, puffy-looking tube is injecting a colorless liquid into its wings. You bid the crew farewell and go your own way. Soon enough, the screen in front of you informs you of the transfer of <payment> from Remnant Logistics, coupled with a short message of appreciation.`
	npc accompany save
		government "Remnant"
		ship "Swan" "Serene Pillar Lost"
		personality escort unconstrained launching

event "ka'het: gaslining probe storm"

mission "Ka'het: Lightning in a Bottle 1"
	landing
	name "Hear from Taely on <planet>"

> 	description "Meet up with Taely on <destination> for an unknown reason."

Reviewer 2: Hmm, I feel like maybe this name and description could use a little work. Maybe "Meet Taely on <planet>" and a little bit better explanation than just that the reason is unknown? I think you have enough text box to work with here that you could maybe mention the gas probe trouble.

Author: "Meet up with Taely on <destination> as her message requested. There appears to be some trouble with their gaslining probes."

	deadline
	deadline 2
	source
		government "Remnant"
	destination "Aventine"
	to offer
		has "event: ka'het: gaslining probe storm"
	on enter

> 		dialog "To your surprise, you hear your commlink chirping once again. It is a message from Taely, asking you to meet up with her in <planet>'s starport as soon as you can. Though she does not specify the reason, the subject of the communication reads 'Gas probe trouble'."

Reviewer 2: ```suggestion
		dialog `To your surprise, you hear your commlink chirping once again. It is a message from Taely, asking you to meet up with her in <planet>'s starport as soon as you can. Though she does not specify the reason, the subject of the communication reads "Gas probe trouble."`
```

American English

> 		dialog `To your surprise, you hear your commlink chirping once again. It is a message from Taely, asking you to meet up with her in <planet>'s starport as soon as you can. Though she does not specify the reason, the subject of the communication reads "Gas probe trouble."`

Reviewer 3: "Starport" is used 38 times in the codebase, but "spaceport" is used over 2000 times.

Is "starport" an error?

Author: Starport is used a number of times in the Remnant story

	on complete
		conversation
			`"We have made a grave mistake," Taely begins, stepping into the <ship>'s hatch. When she turns around, you notice a large, swollen leathery mass peeking behind her neck, almost like an unworn hood. You realize that her movements are slightly jittery, though barely enough to be perceivable. "I am glad you were in this region already, <first>, because we need to act. Evidently, somewhere in our construction process - or even in the design - something slipped beyond our watch."`
			choice
				`	"What do you mean?"`
					goto meaning
				`	"Did the Ka'het notice the Tern probes?"`
				`	"Are you okay?"`
					goto okay
			`	"If they have, it is not a problem right now. I fear the only damage our vessels have incurred was caused by us, rather than the Ka'het."`
				goto meaning
			label okay
			`	"Ah, fairly well," she replies with a hint of perplexity. "Given the issues with the mission, we are all making best use of our flexible hours. More than usual, I admit, but it is a necessity right now."`
			`	"What do you mean with flexible hours?" you ask.`

> 			`	"Sleeping hours." She puts her hand on the back of her head, almost instinctively, before explaining. "One of the more common benefits of our symbionts is the capacity to control better our sleep cycle. Once one stays awake for more than expected in the Circadian rhythm, their body starts being in excess of a number of chemicals, and in deficit of others. Our symbionts are capable of storing the former and administering the latter, at least for a time; then, at the right moment, the process is reversed and we go through the entire sleep we had put off for the previous days.`

Reviewer 2: ```suggestion
			`	"Sleeping hours." She puts her hand on the back of her head, almost instinctively, before explaining. "One of the more common benefits of our symbionts is the capacity to control better our sleep cycle. Once one stays awake for longer than expected by the Circadian rhythm, their body starts being in excess of a number of chemicals, and in deficit of others. Our symbionts are capable of storing the former and administering the latter, at least for a time; then, at the right moment, the process is reversed and we go through the entire sleep we had put off for the previous days.`
```

			choice
				`	"Is there any chance of saving this Tern?"`
				`	"How do I do that?"`
					goto how
			`	"Maybe. If is still able to float in the atmosphere after you are done with it, it might be able to survive through a few more storms before its avionics arc and short-circuit."`
			label how
			`	She hands you a checklist. "I have written down a simple enough sampling procedure for the ion shielding. Still, you will have to send it a command to resurface higher once you are in the system - at the altitude it is staying now, the winds would leave you no chance of rendezvous. And be quick: we cannot know for how long that poor Tern can keep it together in this state."`

mission "Ka'het: Lightning in a Bottle 2"
	landing
	name "Check out the probe in <system>"
	description "Return to <stopover> to take samples from the Tern stationed there."
	source "Aventine"
	stopover "Meligara"
	to offer
		has "Ka'het: Lightning in a Bottle 1: done"
	on stopover
		conversation

> 			`Sitting atop the cloud layer, it takes you a dozen attempts before your signal finally reaches the Tern through the complex atmosphere. Far above, the violet swirl of the wormhole peeks just above the horizon. You wait for the probe to raise through the darkness, wondering how the Remnant can talk with it from so far away.`

Reviewer 2: ```suggestion
			`Sitting atop the cloud layer, it takes you a dozen attempts before your signal finally reaches the Tern through the chaotic atmosphere. Far above, the violet swirl of the wormhole peeks just above the horizon. You wait for the probe to raise through the darkness, wondering how the Remnant can talk with it from so far away.`
```

Maybe?

> 			`Sitting atop the cloud layer, it takes you a dozen attempts before your signal finally reaches the Tern through the chaotic atmosphere. Far above, the violet swirl of the wormhole peeks just above the horizon. You wait for the probe to raise through the darkness, wondering how the Remnant can talk with it from so far away.`

Reviewer 3: "raise through the darkness" -> rise

Author: Fixed

			`	The vehicle shows up upside down, its barrels independently pointing somewhere in the sky. Just as you try an approach, an impressive gust nearly makes you crash; only at the last second the Tern's systems take over, swooshing away from the impact trajectory. Eventually, you can start going through the tasks Taely gave you.`
			choice
				`	(Catch the opportunity to take a closer look to its sensor array.)`
				`	(Go through the checklist and leave.)`
					goto leave

> 			`	The belly of the Tern is adorned with hundreds of small electronic boxes, slightly different from one another but all showing a clear Remnant origin. The dozen or so larger instruments are more interesting. It takes you a while to find an appropriate descriptor of their design language; eventually, you have to settle on 'suspiciously Korath.'`

Reviewer 2: ```suggestion
			`	The belly of the Tern is adorned with hundreds of small electronic boxes, slightly different from one another but all showing a clear Remnant origin. The dozen or so larger instruments are more interesting. It takes you a while to find an appropriate descriptor of their design language; eventually, you have to settle on "suspiciously Korath."`
```

American English.

I'd also check to make sure you're not using single quotes anywhere else for this kind of thing, since I might have missed a couple.

			label leave
			`	Though Taely's handwriting turns out to be uniquely hard to understand, you make it to the end, eventually. A cracking sound from the hull signals you that an ion storm is starting. After a brief moment, you decide to leave the Tern to its fate and watch what happens. You spot tiny blue sparkles all around the atmosphere, briefly illuminating the dark scene. Deep below you, far brighter bolts are barely visible through the clouds. Among them a dark, solid outline suddenly becomes visible.`

> 			`	Your ship's tracker starts whining a desperate whistle. Thousands of signals appear in a matter of seconds, beyond the range of your radar. Their communication is garbled, but your ship tentatively classifies it as 'Builder'. As the storm rages on, you notice on one of your screens the tracking data from the Tern: the pattern of what the Remnant classified as noise is remarkably similar to that of the signals now reaching you.`

Reviewer 2: ```suggestion
			`	Your ship's tracker starts whining a desperate whistle. Thousands of signals appear in a matter of seconds, beyond the range of your radar. Their communication is garbled, but your ship tentatively classifies it as "Builder." As the storm rages on, you notice on one of your screens the tracking data from the Tern: the pattern of what the Remnant classified as noise is remarkably similar to that of the signals now reaching you.`
```

			choice
				`	(Staying was a bad idea. Move out and return to Remnant space.)`
					goto end
				`	(Wait for the storm to end, then leave.)`
					goto end
				`	(Investigate further.)`
			`	A chaotic mix of winds hits the <ship> on the way down, forcing continuous corrections to the descent trajectory. You count three distinct lightning bolts hitting you during these maneuvers, seemingly without ill effects. By the time you are within visual range of one of the mysterious craft, the air is changing so rapidly that the maneuvering engines are essentially always active.`

> 			`	The near-constant lightnings allow you to snap a few low-quality pictures of the strange machine. What appears to be a large balloon sits at the top, its surface often broken up by smaller spikes. A number of tethers hang below it, each holding up machinery of unclear design.`

Author: Hmm, are you sure about this change? I meant that lightnings were constantly appearing and illuminating the sceene

> 			`	The near-constant lightnings allow you to snap a few low-quality pictures of the strange machine. What appears to be a large balloon sits at the top, its surface often broken up by smaller spikes. A number of tethers hang below it, each holding up machinery of unclear design.`

Reviewer 4: The plural of lightning is just lightning, but for the effect you're describing you could say 'lightning flashes'.

> 			`	The near-constant lightnings allow you to snap a few low-quality pictures of the strange machine. What appears to be a large balloon sits at the top, its surface often broken up by smaller spikes. A number of tethers hang below it, each holding up machinery of unclear design.`

Reviewer 4: Although, thinking about it I wouldn't use 'near constant' for that. How about 'incessant' or 'perpetual'?

> 			`	The near-constant lightnings allow you to snap a few low-quality pictures of the strange machine. What appears to be a large balloon sits at the top, its surface often broken up by smaller spikes. A number of tethers hang below it, each holding up machinery of unclear design.`

Author: Changed to "incessant lightning flashes"

			choice
				`	(Wait for the storm to end, then leave.)`
					goto end
				`	(Try to snatch some parts of it.)`
			`	Tremendous gusts move you and your target in every direction, more than once almost sending you into a spin. The first time you arrive next to it, another current arrives to split you apart. The event repeats a second, and a third time. At last, on the fourth attempted approach, the air throws the vessel right into the nose of your ship, crashing before you get the chance to react. Instantly, the balloon pops, and what is left of the machine plunges into the depths of the gas giant.`
			label end
			`	While your ship climbs back toward orbit, you detect a Ka'het attempting to make some runs into the high atmosphere of the gas giant. Though it never descends below the upper layers, it seems to be trying to find a way to reach the Tern.`
	npc
		government "Ka'het"
		system "Ritilas"
		personality staying disables plunders
		fleet
			names "ka'het"
			variant
				"Telis'het (Guns, Disable-able)"
	on complete
		conversation

> 			`Just as you begin to go through the post-landing checks, a light on the console informs you that Taely has already climbed into the cargo bay. Another light soon indicates you that she has opened the mission storage; she seems to have finished going through what you have gathered by the time you reach her.`

Reviewer 4: ```suggestion
			`Just as you begin to go through the post-landing checks, a light on the console informs you that Taely has already climbed into the cargo bay. Another light soon indicates that she has opened the mission storage; she seems to have finished going through what you have gathered by the time you reach her.`
```

			`	She calls the attention of a spaceport worker to ask for a list of expected traffic. She quickly scrolls through the datapad, then turns to you again. "I can ready two Peregrines in the spaceport in a few hours with a soft capture kit. Offhand, I do not believe we can outfit them with quite enough propellant for a round trip to Meligara - at least, not without stopping to ramscoop in the Graveyard. A Swan tanker should solve this, however." After some more napkin math, she signs that she has to go now to be ready in time.`
			choice
				`	"See you in a few hours then."`
					decline
				`	"What about your own probes?"`
			`	"Well," she replies with some trepidation, "they seem to be fine, after all. If nothing new comes up, we can just leave them be on their own mission."`
				decline

mission "Ka'het: Lightning in a Bottle 3A"
	name "Builder balloon"
	description "Escort the Remnant to <destination> to try to capture at least one of the Builder probes deep inside the gas giant."
	source "Aventine"
	destination "Meligara"
	to offer
		has "Ka'het: Lightning in a Bottle 2: done"
	on offer
		event "Ka'het don't spawn in Ritilas"
		conversation

> 			`At the time you arrive, the fleet appears to be made only of two ships - a Peregrine, the back of which is encased in a confusing amount of scaffolding, and a Swan. You are almost hit by a Remnant descending on a rope: once you take a few steps back and glance around, you notice the temporary structure has an extensive pulley system, letting them move easily between different floors. You find Taely busy managing the rest of the Peregrine team.`

Reviewer 2: How is the amount of scaffolding confusing? Is it higher than expected? Lower? Does it just look like a mess?

Author: "confusingly high amount of"?

Reviewer 2: Could work

			choice
				`	"Of course."`
					accept
				`	"What kind of job was the Peregrine on to require such equipment?"`
			`	"Just some scouting in a few nearby systems," she replies, appearing a bit amused. "Best to focus on our own mission for now. Give us just a few more minutes, and then we can begin."`
				accept
	npc accompany save
		government "Remnant"
		ship "Swan" "Serene Pillar Lost"
		ship "Peregrine" "Dark Anchor"
		ship "Peregrine" "Black Song"
		personality escort unconstrained launching

event "Ka'het don't spawn in Ritilas"
	system Ritilas
		remove fleet Ka'het 2700
		remove fleet "Ka'het Aggressive" 9600

event "Ka'het spawn in Ritilas"
	system Ritilas
		add fleet Ka'het 2700
		add fleet "Ka'het Aggressive" 9600

mission "Ka'het: Lightning in a Bottle 3B"
	invisible
	source "Aventine"
	destination "Meligara"
	to offer
		has "Ka'het: Lightning in a Bottle 2: done"
	npc kill
		government "Ka'het"
		system "Ritilas"
		personality staying disables plunders
		fleet
			names "ka'het"
			variant
				"Telis'het (Guns)"
				"Maeri'het (EMP)"

mission "Ka'het: Lightning in a Bottle 4"

> 	name "Success"

Reviewer 2: Will this mission name ever appear to the player? If so it should probably be more descriptive

Author: name "Builder technology to <planet>"
description "Escort the Remnant ships back to <destination> with the hardware taken from the Builder gas probe."

Reviewer 2: Sure, that makes sense

			choice
				`	"Is that one of yours?"`
				`	(Warm up my weapons.)`
			`	Right as the object disappears, a three-letter message finally reaches you through the radio: "E. M. P."`
			`	The electric burst hits you a few seconds later; it seems to have dissipated somewhat, but it is enough to temporarily garble your communicator. A check of the tracking screen shows the Tern unscathed by the strike, but the aft of the Builder probe is now showing a worrying red glow.`
			`	You try to contact the Remnant, but the arrival of another EMP prevents you from doing so. For their part, one of the Peregrines has taken the initiative to approach the alien spacecraft.`
			choice
				`	(Return to orbit to fight the Ka'het.)`
					flee
				`	(Keep trying to speak with the Remnant.)`
			action
				set "ka'het: cloud probe electronics burnt"
			`	The Peregrine pitches up to reveal two enormous claws mounted beneath its body. It closes the distance with the target bit by bit, then makes a swift movement to catch its prey before the wind changes. It succeeds, despite the continuous lightning that now surrounds you.`

> 			`	Once the second Peregrine repeats the movement, a group of Remnant in extravehicular suits come out of their payload bays. Half of them work on securing the lower section of the craft, most of which is now a bright, yellow hue; at the same time, you see the other few distribute along the balloon's various tethers. You see a flash. The balloon soars upward while its payload remains in the clamps of the Peregrine.`

Reviewer 3: "bright, yellow hue" should not have a comma.

Author: Fixed

			`	While you prepare to leave, you give a final look to the tracking screen. At the top, the Ka'het is about to go out of your instruments' range. At the bottom, two sets of debris are falling through the atmosphere: one from the burst balloon, and the other from the Tern.`
				accept

			label "ka'het killed for the cloud probe"
			`Like two enormous birds of prey, the Peregrines have captured the Builder probe with a set of claws mounted beneath their bodies. You receive the communications of a group of Remnant in the middle of an extravehicular activity. Half of them work on securing the lower section of the craft, seemingly having cooled down; at the same time, you see the other few distribute along the balloon's various tethers. You see a flash: the balloon flies away free, while the payload stays in the clamps of the Peregrines.`
			`	"Thank you for getting them off our tail, <first>," Taely radios. "We have what we needed. Plus, while I don't know how long our Tern will keep together, it seems to have survived." The Swan makes a reappearance in your tracking screen, standing well above you in the atmosphere.`

> 			`	"Oddly enough, we have just recorded the Builder balloon implode, rather than burst, once it was free. We were able to catch a few fragments for later study before it fell too deeply into the atmosphere." Soon, the Peregrines signal that they are ready to leave for <planet>.`

Author: It was going up, not down. It was freed up of its payload, so it was a fair bit lighter than before

> 			`	"Oddly enough, we have just recorded the Builder balloon implode, rather than burst, once it was free. We were able to catch a few fragments for later study before it fell too deeply into the atmosphere." Soon, the Peregrines signal that they are ready to leave for <planet>.`

Reviewer 4: Ah, okay, so the part about catching a few fragments was after it had imploded and started to fall again? Maybe in the earlier paragraph which says the balloon 'flies away free' it could explicitly say that it went up? Maybe 'You see a flash. The balloon soars upward while its payload remains in the clamps of the Peregrine.'

> 			`	"Oddly enough, we have just recorded the Builder balloon implode, rather than burst, once it was free. We were able to catch a few fragments for later study before it fell too deeply into the atmosphere." Soon, the Peregrines signal that they are ready to leave for <planet>.`

Author: Changed to that

				accept

			label "ka'het killed early"
			`Gusts only somewhat less violent than the ones you encountered previously surround the Builder vehicle. You notice that the chosen Builder satellite is still in the same area as the Remnant probe, though higher in the atmosphere.`
			scene "scene/builder cloud probe"
			`	Soon enough one of the Peregrines takes the initiative to approach the alien spacecraft. It pitches up to reveal two enormous claws mounted beneath its body. It closes the distance with the target bit by bit, then makes a swift movement to catch its prey before the wind changes. It succeeds.`
			`	You receive the communications of a group of Remnant in the middle of an extravehicular activity. Half of them work on securing the lower section of the craft, seemingly having cooled down; at the same time, you see the other few distribute along the balloon's various tethers. You see a flash: the balloon flies away free, while the payload stays in the clamps of the Peregrines.`

> 			`	"We have what we needed, <first>," Taely states through the ship's radio. "Oddly enough, we have just recorded the Builder balloon implode, rather than burst, once it was free. We were able to catch a few fragments for later study before it fell too deeply into the atmosphere." Soon, the Peregrines signal that they are ready to leave for <planet>.`

Reviewer 4: Again, same question about why it is odd for the balloon to implode.

				accept
	on complete
		event "Ka'het spawn in Ritilas"
		fail "Ka'het: Lightning in a Bottle 3B"
		payment 600000
		conversation
			`Two pairs of camels come out of the Peregrine's payload bay, pulling well-wrapped sections of the Builder craft. Taely and her team follow ceremoniously, staying at the sides of the convoy for the duration of the walk. For its part, the Swan has already taken flight for a new destination.`

> 			`	"I think we can classify this as an overall success." Based on the strange devices in her tool belt, Taely looks ready to start working on the alien vehicle. "I have received communication that all Terns are under attack of the Ka'het EMPs, and most have already faltered. Though the project was not able to complete its original objective, one might say that this discovery is interesting enough to make up for it."`

Reviewer 2: ```suggestion
			`	"I think we can classify this as an overall success." Based on the strange devices in her tool belt, Taely looks ready to start working on the alien vehicle. "I have received communication that all Terns are under attack by Ka'het EMPs, and most have already faltered. Though the project was not able to complete its original objective, one might say that this discovery is interesting enough to make up for it."`
```

			choice
				`	"I can agree with that."`
					goto probe
				`	"Depends from what you can get out of this, I guess."`
					goto probe
				`	"What is going to happen with the program?"`

> 			`	"Evidently, our original premise - that the Ka'het would not be able to reach a gaslining probe - was flawed from the start. If there is one field where the Ka'het show a minimum of inventive, it is direct combat. Arming one of these vehicles with a Point Defense system would have limited benefits, especially compared to starting over with a new idea."`

Reviewer 4: ```suggestion
			`	"Evidently, our original premise - that the Ka'het would not be able to reach a gaslining probe - was flawed from the start. If there is one field where the Ka'het show a modicum of inventiveness, it is direct combat. Arming one of these vehicles with a Point Defense system would have limited benefits, especially compared to starting over with a new idea."`
```

			`	"What kind of idea?" you ask.`
			`	She stops for a bit, then shakes her head. "We will inform you when we get a good one.`
			label probe
			branch "burnt electronics"
				has "ka'het: cloud probe electronics burnt"
			action
				payment 400000
			`	"For now, I can tell you that our hopes for this Builder vehicle look promising. We have found minor signs of damage in some of its avionics, but not from discharge: rather, the electronics appear to have been overcharged in the recent past."`
				goto bye
			label "burnt electronics"
			`	"For now, I can tell you that our hopes for this Builder vehicle look promising. Though most of its avionics show extensive damage, it was not caused by discharge: rather, the electronics appear to have been overcharged in the recent past."`
			label bye

> 			`	Soon enough, the transport stops and Taely's personnel begin unwrapping the previous cargo. "If you find yourself in the southern Graveyard," she says, taking a few steps toward the rest of the team, "I suggest that you keep your eyes open for anything of interest. Our short-lived project could only afford to reach the northern half, after all!" After a farewell, a promise to keep you updated, and the handover of a credit chip, she returns to her own work once again.`

Reviewer 4: ```suggestion
			`	Soon enough, the transport stops and Taely's personnel begin unwrapping the cargo. "If you find yourself in the southern Graveyard," she says, taking a few steps toward the rest of the team, "I suggest that you keep your eyes open for anything of interest. Our short-lived project could only afford to reach the northern half, after all!" After a farewell, a promise to keep you updated, and the handover of a credit chip, she returns to her own work once again.`
```

	npc accompany save
		government "Remnant"
		ship "Swan" "Serene Pillar Lost"
		ship "Peregrine" "Dark Anchor"
		ship "Peregrine" "Black Song"
		personality escort unconstrained launching

mission "Ka'het: Lightning in a Bottle Epilogue 1"
	invisible
	to offer
		has "Ka'het: Lightning in a Bottle 4: done"
		has "Ka'het: Voice From The Past 4: done"
	on offer
		event "Lightning in a Bottle epilogue" 30
		fail

event "Lightning in a Bottle epilogue"

mission "Ka'het: Lightning in a Bottle Epilogue 2"
	invisible
	source
		government "Remnant"
	to offer
		has "event: Lightning in a Bottle epilogue"
	on offer
		outfit "Ka'het Ion Trap" 1

> 		dialog `A message from Taely appears on your commlink. "Captain," it begins, "we have successfully identified the mechanisms capable of generating energy from ion storms in both the cloud probe and the Builder satellite you have helped recover. We have since recovered more examples of this technology; in order to thank you for your efforts, we want to gift you the outfit that was once used by the Fereti warning satellite." A few minutes later, a hurried Tern unloads a compact, spiky-looking device and bolts away.`

Author: This one is actually referring to VFTP. Corrected the rest
